Source/Core/EllipsoidGeometry.js

```diff
@@ -50,6 +50,11 @@ define([
      *
      * @param {Object} [options] Object with the following properties:
      * @param {Cartesian3} [options.radii=Cartesian3(1.0, 1.0, 1.0)] The radii of the ellipsoid in the x, y, and z directions.
+     * @param {Cartesian3} [options.innerRadii=options.radii] The inner radii of the ellipsoid in the x, y, and z directions.
+     * @param {Number} [options.minimumAzimuth=0.0] The minimum azimuth in radians.
```

Comment:
For each new angle parameter here and in `EllipsoidOutlineGeometry`, can the reference doc please include the expected range?  And then throw a `DeveloperError` when the input is out of range?

Reply:
@pjcozzi I'd rather not enforce a specific range, simply because the code can handle any range. It will simply "wrap" the angles if necessary. This also allows you to create some unique shapes if you get creative with the inputs. I'd like to preserve that ability. Let me know if you disagree.

Reply:
OK, just document it please.